### PR TITLE
Update index.md with How-To for NetBeans

### PR DESCRIPTION
**Pull request by 6NGB**
Using Forge with NetBeans. Original method found at https://blogs.oracle.com/geertjan/entry/seamless_minecraft_forge_in_netbeans .
Regards!

### CODE REVIEW

File: docs/gettingstarted/index.md [hunk 1]
```diff
@@ -48,6 +48,16 @@ These instructions assume that you have created the project folder as described
 
 If all the steps worked correctly, you should now be able to choose the Minecraft run tasks from the dropdown, and then click the Run/Debug buttons to test your setup.
 
+Using Forge with NetBeans 8.*
+-----------------------------
+1. Open terminal inside the folder where you extracted Forge MDK/SRC;
+2. Run `gradlew setupDecompWorkspace --refresh-dependencies` and wait for it to finish;
+3. Make sure gradle's task finished properly. If not - consult it's output to fix erors (common issues are firewall and the `JAVA_HOME` environment variable not pointing to a valid JDK);
```
Comment (elifoster) [inline docs/gettingstarted/index.md]: * `If not -` should be `If not,`
* `it's output` should be `its output`
* `erors` should be `errors`

File: docs/gettingstarted/index.md [hunk 2]
```diff
@@ -48,6 +48,16 @@ These instructions assume that you have created the project folder as described
...
+4. Make sure your NetBeans has the _Gradle Support_ plugin, available from NB's _Plugin Portal_. Install it and restart NB if needed (To install a NB plugin, open the IDE and select _Tools > Plugins_. In the _Settings_ tab make sure _Plugin Portal_ is selected);
```
Comment (elifoster) [inline docs/gettingstarted/index.md]: Perhaps link to the plugin

File: docs/gettingstarted/index.md [hunk 3]
```diff
@@ -48,6 +48,16 @@ These instructions assume that you have created the project folder as described
...
+5. Inside NB, shoose _File > Open Project_ and select the Forge MDK/SRC extraction you made earlier. If everything is OK, it'll appear with gradle's circle icon;
```
Comment (elifoster) [inline docs/gettingstarted/index.md]: * `shoose` should be `choose`
* `gradle's` should be `Gradle's`

File: docs/gettingstarted/index.md [hunk 4]
```diff
@@ -48,6 +48,16 @@ These instructions assume that you have created the project folder as described
...
+6. Right click on the Forge project inside the _Projects_ tab. Select _Tasks > run > runClient_. Minecraft will start with Forge enabled. Inside MC you can see the _Example Mod_ listed in the _Mods_ screen;
+7. Enjoy!
```
Comment (elifoster) [inline docs/gettingstarted/index.md]: Possibly unnecessary fluff.

**Review by elifoster — CHANGES_REQUESTED** [review]

**Comment by 6NGB** [comment]
@elifoster Thanks for reviewing. Please check updated version.

**Comment by tterrag1098** [comment]
These instructions duplicate a lot of what is said above. I think they should be mixed in as sub-bullets like the eclipse/IDEA instructions, instead of as a separate section entirely.

**Pull request closed without merging**